_specs/ecip-1048.md

```diff
@@ -4,7 +4,7 @@ ecip: 1048
 title: Clique proof-of-authority consensus protocol
 author: Péter Szilágyi <peterke@gmail.com>
 discussions-to: https://github.com/ethereum/EIPs/issues/225
-status: Draft
+status: Last Call
```

Comment:
```suggestion
status: Final
```

Reply:
I believe we have to take it through "Last Call" to follow the approved process flow, @meowsbits.
Yes, it is a bit silly, but we are playing catch up on "paperwork" which should have been done at the time.

So let's go "Last Call" for two weeks, and then "Accepted" and then "Final".

Following the process.

Reply:
OK.